packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -136,6 +135,21 @@ export default function CommentActionButtons({
           className="btn-tertiary"
         />
       )}
+      {comment.numUpvotes > 0 && (
+        <Button
+          className="btn-tertiary ml-auto"
+          onClick={() => setShowUpvotedPopup(true)}
+        >
+          {comment.numUpvotes} upvotes
+        </Button>
+      )}
+      {showUpvotedPopup && (
```

Comment:
I think it makes more sense to call a callback in the `CommentActionButtons` and to use this modal in the article page component. It feels like this is too much to expect from such a simple component.